packages/shared/lib/core/wallet/stores/main-account-id.store.ts

```diff
@@ -0,0 +1,13 @@
+import { get, writable } from 'svelte/store'
+import { updateActiveWalletPersistedData } from '../../profile'
+import { selectedWalletId } from './selected-wallet-id.store'
+
+export const mainAccountId = writable<string | null>(null)
```

Comment:
will this store load its initial value when the wallet loads, taking the value that is stored in the local storage? For me this value should be a derived store that is derived from the selected wallet